tools/crm_resource.c

```diff
@@ -113,8 +119,13 @@ static gboolean
 resource_ipc_timeout(gpointer data)
 {
     // Start with newline because "Waiting for ..." message doesn't have one
-    fprintf(stderr, "\nAborting because no messages received in %d seconds\n",
-            MESSAGE_TIMEOUT_S);
+    if (error != NULL) {
+        g_clear_error(&error);
+    }
+
+    g_set_error(&error, PCMK__EXITC_ERROR, CRM_EX_TIMEOUT,
+                "Aborting because no messages received in %d seconds", MESSAGE_TIMEOUT_S);
```

Comment:
Need a leading newline (for the reason in the comment)

Reply:
Fixed.